dev/vidify.svg

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Can you use the icon at `vidify/gui/res/icon.svg` instead? Or did you copy it into `dev` because of something?

Reply:
I copied it there because the data_files variable can't rename files (as far as I know), so if I would use `vidify/gui/res/icon.svg` it would install as `/usr/share/pixmaps/icon.svg` (instead of `/usr/share/pixmaps/vidify.svg` which would make the desktop file unable to find it.

Reply:
Makes sense. Can you remove the original `vidify/gui/res/icon.svg`, then? I was saving it there in case I used it inside the app, but it seems that it's useless after all.